### PR TITLE
Khadri 190924 509 pm

### PR DESCRIPTION
**Pull request by manvithagali52**
Added  employee modified files

### CODE REVIEW

**Review by thasmiyas39 — APPROVED** [review]

File: src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeInsertDao.java [hunk 1]
```diff
@@ -2,19 +2,21 @@
...
 import java.sql.Connection;
 import java.sql.PreparedStatement;
+import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.sql.Statement;
 
 import com.khadri.jdbc.prepare.connection.JdbcConnectionUtil;
 import com.khadri.jdbc.prepare.data.employee.dao.model.Employee;
 
 public class EmployeeInsertDao {
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeInsertDao.java]: change class name from EmployeeInsertDao to EmployeeDao
add the select and insert methods into it.

File: src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeSelectDao.java
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeSelectDao.java:1]: Not required this class add the employeeSelectData() into EmployeeDao.java

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java [hunk 2]
```diff
@@ -52,18 +52,36 @@ private void process() {
...
+				EmployeeDataProcessor emp = new EmployeeDataProcessor(scanner, empDao);
+				int empCount = 1;
+				for (int i = 0; i < employeeCount; i++) {
+					emp.process(empCount);
+					empCount++;
+				}
+				System.out.println(employeeCount +"records inserted sucessfully");
+			} else if (operationTypeEmployee == 2) {
+				System.out.println("fetching employee records from the database");
+				EmployeeInsertDao empInsertDao=new EmployeeInsertDao();
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java]: dont create object here , use the instance block initialized empDao reference to call the method.

File: src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeInsertDao.java (hunk 1 above)
Comment (manvithagali52) [reply src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeInsertDao.java]: Resolved

File: src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeSelectDao.java
Comment (manvithagali52) [reply src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeSelectDao.java:1]: I have deleted this class

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java (hunk 2 above)
Comment (manvithagali52) [reply src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java]: yes sir i removed

**Review by sgreddy123 — APPROVED** [review]

File: src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeSelectDao.java
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeSelectDao.java:1]: why you need this file?

**Review by khadri2021 — APPROVED** [review]

**Pull request merged**